src/main/webapp/WEB-INF/views/car/addCar.jsp

```diff
@@ -0,0 +1,15 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+  <title>Add Car</title>
+</head>
+<body>
+<h1>Fill the form below</h1>
+
+<form method="post" action="${pageContext.request.contextPath}/cars/add">
+  Model <input type="text" name="model"><br>
```

Comment:
`<input type="text" name="model" required>`
If you want, you can make all input fields required to fill, to avoid cases with empty fields. But it is not necessary, in my opinion